README.md

```diff
@@ -68,13 +68,22 @@ final class AppDelegate: ApplicationDelegate {
 		sudo apt update
 		sudo apt install vulkan-sdk -y
 		```
-	- Alternativelly you can try to install Vulkan SDK from tarballs provided by LunarG or if you are using Debian 12 and newer or Ubuntu 22.04 and newer you can substitute Vulkan SDK from LunarG by installing 	libvulkan-dev
+	- If you are using Debian 12+ or Ubuntu 22.04 and newer you can substitute Vulkan SDK from LunarG by installing libvulkan-dev
```

Comment:
I think naming consistency is important. Either say "Debian 12+ or Ubuntu 22.04+" or leave it as it was please

Reply:
done